ConfigSpace/hyperparameters.pyx

```diff
@@ -26,19 +26,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import copy
+import io
 # cython: language_level=3
-
-from abc import ABCMeta, abstractmethod
+import math
 import warnings
-
-from ConfigSpace.hyperparameters cimport Hyperparameter
-
 from collections import OrderedDict, Counter
-import copy
 from typing import List, Any, Dict, Union, Set, Tuple, Optional
-import io
-import numpy as np
+
 cimport numpy as np
+import numpy as np
```

Comment:
Is there a reason to swap these around?

Reply:
I think this was just done by auto-formatting, so there's no special reason behind it.